.travis.yml

```diff
@@ -1,4 +1,3 @@
-bundler_args: --without doc:util:site:benchmark:development
```

Comment:
I thought maybe that too, I'm gonna play with it in the morning and see how it broke it in the Travis-CI image.